[Browser log] add checks on window.DD_LOGS (#5447)
DataDog/documentation: content/en/logs/log_collection/javascript.md

* `http.useragent`
* `network.client.ip`

> **Note:** `window.DD_LOGS` check is added to prevent any issues in case of a loading failure of the library.

Reviewer: ```suggestion
**Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the library.
```

## Advanced usage

### Filter by status

In some cases, you might want to disable the debug mode or to only collect warnings and errors. This can be achieved by changing the logging level: set the `level` parameter to `debug` (default), `info`, `warn`, or `error` :

```
window.DD_LOGS && DD_LOGS.logger.setLevel('<LEVEL>')
```

Only logs with a status equal to or higher than the specified level are sent.

> **Note:** `window.DD_LOGS` check is added to prevent any issues in case of a loading failure of the library.

Reviewer: ```suggestion
**Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the library.
```

### Change the destination

By default, the loggers are sending logs to Datadog. It is also possible to configure the logger to send logs to the console, or to not send logs at all. This can be used in the development environment to keep the logs locally.

Use the `setHandler` function with the values `http` (default), `console`, or `silent`:
```
window.DD_LOGS && DD_LOGS.logger.setHandler('<HANDLER>')
```

> **Note:** `window.DD_LOGS` check is added to prevent any issues in case of a loading failure of the library.

Reviewer: ```suggestion
**Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the library.
```

...
<script>
...
if (window.DD_LOGS) {
    const signupLogger = DD_LOGS.getLogger('signupLogger')
    signupLogger.info('Test sign up completed')
}
...
</script>
...
```

> **Note:** `window.DD_LOGS` check is added to prevent any issues in case of a loading failure of the library.

Reviewer: ```suggestion
**Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the library.
```

### Overwrite context

It is possible to set the entire context in one call. This also overrides previously set attributes, if any:

```
# For one logger
if (window.DD_LOGS) {
    my_logger.setContext(<JSON_ATTRIBUTES>)
}

# For the global context
window.DD_LOGS && DD_LOGS.setLoggerGlobalContext(<JSON_ATTRIBUTES>)
```

**Example:**

```
if (window.DD_LOGS) {
    const signupLogger = DD_LOGS.getLogger('signupLogger')
    signupLogger.setContext({
      env: 'staging',
      team: 'user-account'
    })
}
```

> **Note:** `window.DD_LOGS` check is added to prevent any issues in case of a loading failure of the library.

Reviewer: ```suggestion
**Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the library.
```
